packages/js/plugins/http/schema.graphql

```diff
@@ -9,30 +9,43 @@ type UrlParam {
 }
 
 type Response {
-  status: Int!
+  status: UInt64!
   statusText: String!
   headers: [Header!]
-  body: String
+  body: String!
```

Comment:
Do you think we should make `JSON` a base type, just like we made `Bytes` a supported base type in GraphQL?

Reply:
Also, is this always present? Can responses have a null body? If so we can make this optional.

Reply:
> Do you think we should make `JSON` a base type, just like we made `Bytes` a supported base type in GraphQL?

It would be convenient here for sure but I am not sure if there are additional use-cases for this - if we think there will be additional use-cases then we should implement `JSON` base type for sure